ql/src/test/results/clientpositive/llap/vectorized_murmurhash_repeating.q.out

```diff
@@ -0,0 +1,8374 @@
+PREHOOK: query: create table csv_table(id string, id2 string) row format delimited fields terminated by ',' stored as textfile
+PREHOOK: type: CREATETABLE
+PREHOOK: Output: database:default
+PREHOOK: Output: default@csv_table
+POSTHOOK: query: create table csv_table(id string, id2 string) row format delimited fields terminated by ',' stored as textfile
+POSTHOOK: type: CREATETABLE
+POSTHOOK: Output: database:default
+POSTHOOK: Output: default@csv_table
+PREHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/2048.csv' OVERWRITE INTO TABLE csv_table
+PREHOOK: type: LOAD
+#### A masked pattern was here ####
+PREHOOK: Output: default@csv_table
+POSTHOOK: query: LOAD DATA LOCAL INPATH '../../data/files/2048.csv' OVERWRITE INTO TABLE csv_table
+POSTHOOK: type: LOAD
+#### A masked pattern was here ####
+POSTHOOK: Output: default@csv_table
+PREHOOK: query: CREATE TABLE source(id string, id2 string) stored as orc
+PREHOOK: type: CREATETABLE
+PREHOOK: Output: database:default
+PREHOOK: Output: default@source
+POSTHOOK: query: CREATE TABLE source(id string, id2 string) stored as orc
+POSTHOOK: type: CREATETABLE
+POSTHOOK: Output: database:default
+POSTHOOK: Output: default@source
+PREHOOK: query: insert overwrite table source select id, id2 from csv_table
+PREHOOK: type: QUERY
+PREHOOK: Input: default@csv_table
+PREHOOK: Output: default@source
+POSTHOOK: query: insert overwrite table source select id, id2 from csv_table
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@csv_table
+POSTHOOK: Output: default@source
+POSTHOOK: Lineage: source.id SIMPLE [(csv_table)csv_table.FieldSchema(name:id, type:string, comment:null), ]
+POSTHOOK: Lineage: source.id2 SIMPLE [(csv_table)csv_table.FieldSchema(name:id2, type:string, comment:null), ]
+PREHOOK: query: explain vectorization detail
+select murmur_hash(id, id2) from source
+PREHOOK: type: QUERY
+PREHOOK: Input: default@source
+#### A masked pattern was here ####
+POSTHOOK: query: explain vectorization detail
+select murmur_hash(id, id2) from source
+POSTHOOK: type: QUERY
+POSTHOOK: Input: default@source
+#### A masked pattern was here ####
+PLAN VECTORIZATION:
+  enabled: true
+  enabledConditionsMet: [hive.vectorized.execution.enabled IS true]
+
+STAGE DEPENDENCIES:
+  Stage-0 is a root stage
+
+STAGE PLANS:
+  Stage: Stage-0
+    Fetch Operator
+      limit: -1
+      Processor Tree:
+        TableScan
+          alias: source
+          Select Operator
+            expressions: hash(id,id2) (type: int)
```

Comment:
In this and all subsequent explains we can see that the vectorized murmur implementation is not used. In other words, the tests here do not cover the problematic NPE case that was reported. This can be easily seen since this test pass cleanly on apache master without the changes in this PR.

Reply:
Sorry for the wrong test; I thought `sum()` is irrelevant to qtests so I missed checking the qtest output. It seems that we need `sum()` to enforce vectorization and using `murmur_hash()`. I reverted the query to use sum().